docs/docs/_available-tags.md

```diff
@@ -0,0 +1,29 @@
+# Available tags
```

Comment:
Are we using this file anywhere? I can't seem to find any reference to it.

Reply:
No that's correct. I'd thought it best to reference it in a wiki page where we describe how to contribute. We could also add a reference from the central tags md though. (The one I'll merge with index.md)